torch/quantization/observer.py

```diff
@@ -649,10 +651,26 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                     self.min_vals.resize_(val.shape)
                 else:
                     self.max_vals.resize_(val.shape)
+                # For torchscript module we need to update the attributes here since we do not
+                # call the `_load_from_state_dict` function defined module.py
+                if torch.jit.is_scripting():
+                    if name == 'min_vals':
+                        self.min_vals = val
```

Comment:
can this be `self.min_vals.copy_(val)`, to preserve the dtype and device of the source?